policyengine_us/variables/gov/states/az/hhs/tanf/eligibility/az_fpg_baseline.py

```diff
@@ -0,0 +1,20 @@
+from policyengine_us.model_api import *
+
+
+class az_fpg_baseline(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "Arizona Cash Assistance Payment Standard fpg Baseline"
+    definition_period = YEAR
+    reference = "https://des.az.gov/services/child-and-family/cash-assistance/cash-assistance-ca-income-eligibility-guidelines"
+    defined_for = "az_hhs_tanf_eligible"
+
+    def formula(spm_unit, period, parameters):
+        fpg_year = f"1992-01-01"
```

Comment:
By convention for constants
```suggestion
        FPG_YEAR = f"1992-01-01"
```